include/entity.h

```diff
@@ -1986,6 +1986,21 @@ typedef struct {
     /* 0xAE */ s16 unkAE;
 } ET_BackgroundVortex;
 
+typedef struct {
+    s32 pad0[1];
```

Comment:
For pads let's use the "s32 : 32" syntax like you'll see on other entities. Makes it easy to tell what's a pad and what isn't, without needing to name all your pads. Struct offset comments would also be nice.